src/core.js

```diff
@@ -49,7 +49,13 @@ function createModule( name, testEnvironment ) {
 		testsRun: 0,
 		unskippedTestsRun: 0,
 		childModules: [],
-		suiteReport: new SuiteReport( name, parentSuite )
+		suiteReport: new SuiteReport( name, parentSuite ),
+
+		// Pass along `skip` property from parent module, in case there is one,
+		// to childs. And use own otherwise.
+		// This property will be used to mark own tests and tests of child suites
+		// as `skipped`.
+		skip: parentModule !== null && parentModule.skip || skip
```

Comment:
Just crossed my mind right now. What should be the order here ?
Should the modifier of the current module take precedence over the one from the parent module ?

One thing I forgot to test is what will happen if we defined an `only` module inside a `skipped/todo` one (I know this is rarely going to happen), or a `skipped` test inside a `todo` and vise versa ?